docs/source/migrations/1.6.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Apollo iOS 1.6 migration guide
+description: From 1.5 to 1.6
+---
+
+This guide describes the process of migrating your code from version 1.5 to version 1.6 of Apollo iOS. Please follow the relevant migration guides if you're on a version other than 1.5.
+
+## Apollo Codegen SPM Package
+
+The 1.6 update restructured the Apollo iOS ecosystem by splitting the code up into multiple different repositories/packages. One of these changes was the creation of the new `apollo-ios-codegen` repo and SPM package for users who want to do their code generation through Swift vs using the CLI tool.
+
+> **Note:** For more information on the project restructuring in the 1.6 release see [this](https://github.com/apollographql/apollo-ios/issues/3240) GitHub issue.
+
+If you are doing your code generation through Swift you have something like the following in your `Package.swift` file:
```

Comment:
It's good that we specifically mention "code generation through Swift" but would it be worth calling out that CLI codegen users are unaffected?